src/partial/6_pricing-section/_pricing-section.html

```diff
@@ -0,0 +1,22 @@
+<section>
+    <div class="container pricing_container">
+        <div class="pricing_title">
+            Pricing Plans
+        </div>
+        <div class="pricing_description">
+            Lorem ipsum dolor sit amet, pro aeterno adipisci ex, pro no iriure accusam. Eros admodum intellegat ex mea,mei wisi nullam ne, qui cu aliquid nominavi. Est habemus maiestatis ut. Vim quod elitr interesset ea.
+        </div>
+        <div class="pricing_prices-wrapper">
+            <!-- partial:cards/_basic-card.html -->
+            <!-- partial -->
+            <!-- partial:cards/_standard-card.html -->
+            <!-- partial -->
+            <br class="hidden-xs hidden-lg">
```

Comment:
I noticed that this `br` is being used yo layout purposes. What can you do to remove this and use only CSS?

Reply:
Here a grid layout such as Bootstrap's `row` and `col` layout would be able to remove the `<br>` tag and use only CSS.

```html
<div class="row">
    <div class="col-xs-12 col-sm-6 col-md-3">
        <!-- partial:cards/_basic-card.html -->
        <!-- partial -->
    </div>
    <div class="col-xs-12 col-sm-6 col-md-3">
        <!-- partial:cards/_standard-card.html -->
        <!-- partial -->
    </div>
    <div class="col-xs-12 col-sm-6 col-md-3">
        <!-- partial:cards/_deluxe-card.html -->
        <!-- partial -->
    </div>
    <div class="col-xs-12 col-sm-6 col-md-3">
        <!-- partial:cards/_unlimited-card.html -->
        <!-- partial -->
    </div>
</div>
```

This would be a good start, but since Bootstrap's row and columns alter the margin and padding, there would be extra work in fixing the `pricing_card` class.